tests/Smoke/CiIntegrationTest.php

```diff
@@ -180,7 +182,7 @@ public function provideIntegrationCases()
                     'dir b/file b.php',
                     '',
                 ),
-                'S.',
+                '(?:S[.]|[.]S)',
```

Comment:
this approach will be problematic when we will have case not with 2 files processed, but with more, like 10, 20, etc....

Reply:
Indeed it may, but that would also imply to make up ones mind about what the future might bring and as that is full of many different possibilities and outcomes I personally would not deem it so fitting for a fix and development discussion. In the concrete scenario of the fixture this is with two files (in the same order at iteration but different order at output).

Reply:
even with having 5 files only (like 3x`.` and 2x`S`) this is gonna be more and more complex.
and even now, with just 2 files, data providers in this PR are more complex than original code.